eth/backend.go

```diff
@@ -222,14 +221,16 @@ func New(stack *node.Node, config *ethconfig.Config) (*Ethereum, error) {
 	if config.BlobPool.Datadir != "" {
 		config.BlobPool.Datadir = stack.ResolvePath(config.BlobPool.Datadir)
 	}
-	blobPool := blobpool.New(config.BlobPool, eth.blockchain)
+	// CHANGE(TAIKO): disable blob pool
+	//blobPool := blobpool.New(config.BlobPool, eth.blockchain)
```

Comment:
Do we need to do that? since our chain is just on Shanhgai fork?

Reply:
Actually, shanghai fork will return an error. Reverted.